src/core/icons/fileIcons.ts

```diff
@@ -2607,5 +2607,9 @@ export const fileIcons: FileIcons = {
       name: 'gemini-ai',
       fileNames: ['.aiexclude'],
     },
+    {
+      name: 'taze',
+      fileNames: ['taze.config.js'],
```

Comment:
Could you please also add a "ts" file name here?

I just searched for it and found a repo where somebody is using a ts file instead of js:
https://github.com/antfu-collective/ni/blob/main/taze.config.ts

Reply:
Later

Reply:
> Could you please also add a "ts" file name here?
> 
> I just searched for it and found a repo where somebody is using a ts file instead of js:
> https://github.com/antfu-collective/ni/blob/main/taze.config.ts

Do it yourself, add all the JS file extensions (ts, mjs, cjs), I'm just too busy right now

Reply:
I'll do it :)